app/src/main/AndroidManifest.xml

```
@@ -1,18 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="com.innovatube.boilerplate">
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
 
     <application
         android:name=".InnovatubeApplication"
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        tools:ignore="GoogleAppIndexingWarning">
```

Please provide reason for `GoogleAppIndexingWarning`

---

This is lint warning
```App is not indexable by Google Search; consider adding at least one Activity with an ACTION-VIEW intent-filler. See issue explanation for more details.```
I want to disable this lint warning

---

Check here please: http://java-lang-programming.com/en/lints/1

---

I know. It's only another way

---

Nope, the difference is you have to add `tools:ignore` for every lint warning (luckily this example only have one but I don't like the way of workaround). So moving the config to gradle or create a base `lint-baseline.xml` and define the config insde

---

Yes, It has only one. At least I know where this lint is checked. If you add tools:ignore="SmallSp" to the gradle or create a base `lint-baseline.xml` you said. I will add it.
too many `tools:ignore` in `item_header.xml`, `item_home_header.xml`, `item_highlight.xml`...

---

You could handle it too @imtuann moving to `lint-baseline.xml` is a good move

---

I assigned it to you @imtuann

---

We need those warning. We should know where we're wrong and where lint is checked. Please ignore this issue.

---

It is configurable, It means the warnings were well aware of and were skipped by business purpose (not silently ignore)

---

@imtuann : Following here to create `lint-baseline.xml`. It is the best option for now and It could be used later when we need to custom the lint rule. Also please consider to define base rule for `ktlint`